### PR TITLE
init: add page

### PR DESCRIPTION
**Pull request by jangidkrishna**
----

- [x] The page (if new), does not already exist in the repo.

- [x] The page (if new), has been added to the correct platform folder:  
      `common/` if it's common to all platforms, `linux/` if it's Linux-specific, and so on.

- [x] The page has 8 or fewer examples.

- [x] The PR is appropriately titled:  
      `<command name>: add page` for new pages, or `<command name>: <description of changes>` for pages being edited

- [x] The page follows the [contributing](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md) guidelines

### CODE REVIEW

File: pages/linux/init.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# init
+
+> Its primary role is to create processes from a script stored in the file /etc/inittab.
```
Comment (sbrl) [inline pages/linux/init.md]: `primary role`? This implies that it does other things besides the thing you describe here. Perhaps rewording this a bit would help improve clarity? The description should give a short, concise description of what the command does.

File: pages/linux/init.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+- Shutsdown the system:
```
Comment (sbrl) [inline pages/linux/init.md]: `Shutsdown` -> `Shutdown`

File: pages/linux/init.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+`init 0`
+
+- Single user mode or emergency mode:
```
Comment (sbrl) [inline pages/linux/init.md]: Perhaps `Enter single user mode or emergency mode` would read better here?

File: pages/linux/init.md [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+
+- Reboot the system:
+
+`init 6`
+
+- Boot directly into a single user shell:
+
+`init -b`
+
+- Tell the init command to enter the maintenance mode:
```
Comment (sbrl) [inline pages/linux/init.md]: Perhaps just `Enter maintenance mode` would be appropriate here?

**Comment by jangidkrishna** [comment]
i have done the changes thanks for the help

File: pages/linux/init.md [hunk 5]
```diff
@@ -0,0 +1,23 @@
+# init
+
+> Init is a daemon process that start during booting continue to run until the system is shut down.
```
Comment (agnivade) [inline pages/linux/init.md]: No need to mention "Init" in the sentence. It is expected that the main heading will be about the tool itself. I recommend a slight change - 
```
A daemon, which is the first process to be started by the kernel, and continues to run until the system is shut down.
```

File: pages/linux/init.md [hunk 6]
```diff
@@ -0,0 +1,23 @@
...
+
+- Shutdown the system:
```
Comment (agnivade) [inline pages/linux/init.md]: => "Shut down"

File: pages/linux/init.md [hunk 7]
```diff
@@ -0,0 +1,23 @@
...
+
+`init 1`
+
+- Reboot the system:
+
+`init 6`
+
+- Boot directly into a single user shell:
+
+`init -b`
```
Comment (agnivade) [inline pages/linux/init.md]: Is the `-` required ? If so, is it not required for the maintenance mode ?

File: pages/linux/init.md (hunk 6 above)
Comment (sbrl) [reply pages/linux/init.md]: Good point! I guess because I say / write `shutdown`, I never though to think of the proper / correct form here

**Comment by agnivade** [comment]
@jangidkrishna - any updates on this ?

**Pull request closed without merging**